packages/design-system/src/components/Pagination/Pagination.tsx

```diff
@@ -0,0 +1,239 @@
+import Ellipses from './Ellipses'
+import Page from './Page'
+import React from 'react'
+import classNames from 'classnames';
+
+export interface PaginationProps {
+  /**
+   * Class to be applied to parent `<nav>` element of Pagination component. Optional.
+   */
+  className?: string,
+  /**
+   * Renders compact layout. Optional.
+   */
+  compact?: boolean,
+  /**
+   * Defines initial active page in Pagination. Optional. 
```

Comment:
[suggestion] Consider updating this to `Defines active page in Pagination. Optional` since the component is not stateless and there is no "initial" active page

Reply:
Like it - changing!